specutils/spectra/spectrum_collection.py

```diff
@@ -0,0 +1,90 @@
+import logging
+from collections import MutableSequence
+
+import astropy.units as u
+import numpy as np
+from astropy.nddata import NDIOMixin, NDUncertainty
+
+from .spectrum1d import Spectrum1D
+
+__all__ = ['SpectrumCollection']
+
+
+class SpectrumCollection(MutableSequence, NDIOMixin):
+    """
+    A container class for :class:`~specutils.Spectrum1D` objects. This allows 
+    for operations to be performed over a set of spectrum objects. This class 
+    behaves as if it were a :class:`~specutils.Spectrum1D` and supports
+    pythonic collection operations like slicing, deleting, and inserting.
+
+    Note
```

Comment:
I *think* this has to be "Notes" (with the s) to get picked up as the correct section by sphinx/numpydocs?